source/_integrations/unifiprotect.markdown

```diff
@@ -86,6 +83,16 @@ You can replace `2.0.0-beta.5` with whatever version of UniFi Protect you want t
 
 **Note**: if you want to downgrade to another Early Access version, you must have <a href="https://help.ui.com/hc/en-us/articles/115012240067-UniFi-How-to-enable-remote-access" target="_blank" rel="noopener">Remote Access enabled</a> and have the Early Access release channel enabled.
 
+#### Using Early Access Releases
+
+If you have chosen to ignore all of the warnings about Early Access, here are some rules/tips to follow when using Early Access releases.
+
+* Report issues [upstream](https://github.com/AngellusMortis/pyunifiprotect/issues/) rather to Home Assistant Core
```

Comment:
We should not point users to external issue trackers. It is already hard enough for users to find their way around the GitHub space.

Additionally, Home Assistant provides links to view open issue for an integration in several places, which render useless with these practices.

Please open a PR to remove this part.